helpers_test.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	shutil "github.com/termie/go-shutil"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func commitAllChanges() {
+	err := run([]string{"git", "add", "-A"})
```

Comment:
Use a variadic function to make this interface nicer:

```go
err := run("git", "add", "-A")
```

Example implementation at 
https://github.com/Originate/git-town/blob/master/src/util/util.go#L45